src/integrations/stripe.ts

```diff
@@ -33,25 +34,25 @@ export function installStripe(stripeConfig: any) {
 // Returns mapping of servar key (or hidden field key) to value
 export function getFlatStripeCustomerFieldValues(
   stripeConfig: any,
-  fieldValues: any
+  _fieldValues: any
```

Comment:
instead of passing fieldValues in everywhere, can you use it directly from the import?

Reply:
Doing it this way makes it convenient for test cases.  I don't actually pass it at runtime and it does use the import at runtime.  It is only passed from the test case.

Reply:
i would prefer we don't structure production code around test case convenience. You can always initialize fieldValues at a global level (we do that for all other test cases)

Reply:
Changes made.